.env.example

```diff
@@ -0,0 +1,20 @@
+# Logging
+DEBUG=info*,error*,warn*,debug*
```

Comment:
Oh actually one note -- Does setting the DEBUG flag here work for you? For me it doesn't actually print anything out & I think I've only gotten it working with `env DEBUG="info*,error*,warn*,debug*" yarn start`

Reply:
It really should work like this though & generally I really wanna update our logging library

Reply:
the flag here worked for me, but i also ran `export DEBUG=info*,error*,warn*,debug*` so will try on a fresh terminal

Reply:
both with and without the quotes work for me!